maps/src/main/java/org/odk/collect/maps/MapPoint.java

```diff
@@ -20,9 +20,9 @@
 import java.util.Locale;
 
 public class MapPoint implements Parcelable {
```

Comment:
I think you can convert this class to kotlin and then remove the code responsible for making it Parcelable see https://developer.android.com/kotlin/parcelize